src/components/Products.jsx

```diff
@@ -26,12 +43,29 @@ const Products = () => {
           <Item.Header>{product.name}</Item.Header>
           <Item.Meta> {`${product.price}kr`}</Item.Meta>
           <Item.Description>{product.description}</Item.Description>
+          <button
+          data-cy="order-button"
+          onClick={() => addToOrder(product.id)}
+          className="ui button"
+        >
+          order +
+        </button>
         </Item.Content>
       </Card>
-    );
-  });
+  
+  );
+    
+});
+  return(
+  <>
+  <Container>
+    <List inverted id="products-list" size="big">
+      {productlist}
+    </List>
+  </Container>
+  <ToastContainer data-cy="message-box" />
+</>
```

Comment:
Please get into the habit of formatting your code.

Reply:
no problem. thank you for mention!